client/lib/gsuite/constants.js

```diff
@@ -13,3 +13,12 @@ export const GSUITE_SLUG_PROP_TYPES = PropTypes.oneOf( [
 ] );
 
 export const GSUITE_LINK_PREFIX = 'https://mail.google.com/a/';
+
+/**
+ * We'll use the following constants to do the switchover from
+ * G Suite -> Google Workspace.
+ * The product name is never translated in the translation strings
+ * so we won't translate them here either.
+ */
+export const GSUITE_PRODUCT_NAME = 'G Suite';
+export const GOOGLE_WORKSPACE_PRODUCT_NAME = 'Google Workspace';
```

Comment:
I'd like to suggest different names for those constants. Once we start selling Google Workspace, we'll indeed have to deal with the following Store products:

* G Suite Basic - although referenced as G Suite in the Store and other places (which was not a great idea)
* G Suite Business
* Google Workspace Business Starter

So I see `G Suite` and `Google Workspace` more like the names of product families, and `Google Workspace Business Starter` as a product name. We could be more consistent with the `getGoogleMailServiceName()` function as well, which as far I can see is the only piece of code using those constants. So maybe `GOOGLE_MAIL_SERVICE_GSUITE` and `GOOGLE_MAIL_SERVICE_GOOGLE_WORKSPACE` instead?